src/components/cross-chain.js

```diff
@@ -92,14 +90,17 @@ class CrossChainRecoveryForm extends Component {
       prv
     } = this.state;
 
+    const sourceCoin = (bitgo.env === 'test') ? 't' + this.state.sourceCoin : this.state.sourceCoin;
```

Comment:
Actually, I would default to the testnet coin, and only use the production coin if bitgo.env === 'prod' (same for next line)